lib/cartopy/io/shapereader.py

```diff
@@ -386,7 +389,7 @@ def default_downloader():
 ne_{resolution}_{name}.shp
 
         """
-        default_spec = ('shapefiles', 'natural_earth', '{category}',
+        default_spec = ('shapefiles', 'natural_earth', '{category}', '{version}',
```

Comment:
Looks like the doctest failure is related to the docstring right above this. Where it got a `/{version}/` added in now. I actually don't think that should be included there now because it is path joining the version, but we don't want that if version is `None`, we want to leave off the `/`. I guess the question is does this need to be udpated since it is the default case (no version) or can we just leave it alone for now?